docs/content/docs/try-flink-ml/java/quick-start.md

```diff
@@ -73,20 +72,13 @@ initialization.
 
 {{< stable >}}
 
-Please [download the correponding binary
-release](https://flink.apache.org/downloads.html) of Flink ML, then extract the
-archive:
+Please download [Flink ML Python
+source](https://flink.apache.org/downloads.html) and extract the jar files into
```

Comment:
I didn't find the apache-flink-ml*.tar.gz on this website, only source file [flink-ml-2.2.0-src.tgz](https://dlcdn.apache.org/flink/flink-ml-2.2.0/flink-ml-2.2.0-src.tgz) is available, which doesn't contains ".../deps/lib/*.jar"

Reply:
The link for Flink ML python source will be available after https://github.com/apache/flink-web/pull/659 is merged.

It will be copied from https://dist.apache.org/repos/dist/dev/flink/flink-ml-2.3.0-rc1/.